runtime/tr.source/trj9/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -13004,6 +13004,8 @@ inlineAtomicStampedReference_doubleWordSet(
    return true;
    }
 
+// Note that this function must have behaviour consistent with the OMR function
```

Comment:
This should either be properly doxygen'd or the comment moved into the method body and not put in the place where doxygen would go.